common/app-starters-micrometer-common/src/test/resources/org/springframework/cloud/stream/app/micrometer/common/cloud-test-info.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "sso",
+  "label": "sso",
+  "plan": "notfree",
+  "tags": ["configuration"],
+  "credentials":{
+    "uri": "http://pivotal.io",
+    "client_id": "fakeClientId",
+    "client_secret": "fakeSecret",
+    "access_token_uri": "token"
+  }
+}
```

Comment:
See my comment above for the json content.

Reply:
@sabbyanandan i've submitted a PR on top of your PR ;)(https://github.com/sobychacko/core/pull/1) that applies the above-suggested changes. 

Or if you are fine with those changes i can apply them and merge it my self?

Reply:
@tzolov Please go ahead with your changes and suggestions and merge. (The only reason I put that new artifact as `test` scope in the BOM is that it is already a test artifact and doesn't contain any non-test code, but please go with what you said for consistency reasons)